.github/workflows/tester.yml

```
@@ -0,0 +1,60 @@
+name: Tester
+
+on: [push, pull_request]
+
+jobs:
+  tester:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        node-version: ['12.x', '14.x', '16.x']
+        HEXO_VERSION: ['3.x', '4.x', '5.x']
+        
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Cache NPM dependencies
+        uses: actions/cache@v2
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-npm-cache
+          restore-keys: ${{ runner.os }}-npm-cache
+      - name: Install Dependencies
+        run: yarn install && yarn add "hexo@${{ matrix.HEXO_VERSION }}"
+      - name: Test
+        run: yarn test
+        env:
+          CI: true
+  coverage:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node-version: ['14.x']
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Cache NPM dependencies
+        uses: actions/cache@v2
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-npm-cache
+          restore-keys: ${{ runner.os }}-npm-cache
+      - name: Install Dependencies
+        run: yarn install && yarn add "hexo@5.x"
+      - name: Coverage
+        run: yarn test-cov
+        env:
+          CI: true
+      - name: Coveralls
+        uses: coverallsapp/github-action@master
+        with:
+          github-token: ${{ secrets.github_token }}
```

Is the lower case version accepted? I thought that it must be upper case 
```suggestion
          github-token: ${{ secrets.GITHUB_TOKEN }}
```

---

Seems to also work in lowercase https://coveralls.io/github/tomap/hexo-fontawesome